mkdocs.yml_template

```diff
@@ -30,6 +30,12 @@ nav:
         - "5 - Python expressions in properties": tutorials/articles/understanding_gui/step_05/step_05.md
         - "6 - Page layouts": tutorials/articles/understanding_gui/step_06/step_06.md
         - "7 - Multi-pages, navbars, and menus": tutorials/articles/understanding_gui/step_07/step_07.md
```

Comment:
I thought the purpose was to replace the "Understanding GUI" tutorial. Here you just added a new one.

Reply:
Yeah, I was not sure if you guys would agree to replace the "Understanding GUI" tutorial so for the time I put it separately. When I'm done with the requested changes and if everyone agrees, this tutorial will replace "Understanding GUI"